main.go

```diff
@@ -12,11 +13,9 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-
 	utils.SetupRootCommand(rootCmd)
-
 	err = rootCmd.Execute()
 	if err != nil {
-		log.Fatal(err)
```

Comment:
Removed this log entry since the `Execute` method already outputs the error.
Previously, the error was outputted twice.